src/components/Button/Button.tsx

```diff
@@ -1,7 +1,8 @@
 import React, { FC } from 'react';
 import classNames from 'classnames';
+import styles from './Button.module.scss';
```

Comment:
Imports are another area where it would be good to standardize how they're organized. I generally do: 3rd party imports, then internal component imports, then css/scss. So this block would be:

```
import React from 'react';
import PropTypes from 'prop-types';
import classNames from 'classnames';
import Spinner from '../Spinner/Spinner';
import './Button.scss';	import styles from './Button.module.scss';
```
Or, we could even add spaces between the groups to distinguish them even more. I generally only add a space between the js and css; so: 

```
import React from 'react';
import PropTypes from 'prop-types';
import classNames from 'classnames';
import Spinner from '../Spinner/Spinner';

import './Button.scss';	import styles from './Button.module.scss';
```

Or, we could add a space between each:

```
import React from 'react';
import PropTypes from 'prop-types';
import classNames from 'classnames';	

import Spinner from '../Spinner/Spinner';

import './Button.scss';	import styles from './Button.module.scss';
```

Reply:
We really should settle on a pattern for this. Until then, can we have the style imports come after the JS?

I've created an issue to standardize this across the repo: #127